package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint --fix --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "lint": "eslint --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "pretest": "rimraf ./.nyc_output",
-    "test": "cross-env NODE_ENV=test nyc mocha --recursive --require @babel/register test",
+    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test.*'",
```

Comment:
From what I could tell, the `cross-env` and `nyc` dependencies weren't providing any meaningful functionality. My observations seemed to indicate that these two were functionally identical.